docs/push-and-poll-events.md

```diff
@@ -0,0 +1,19 @@
+# SSE Communication Sequence
+SSE (Shared Signal Events) has two options to deliver SETs (Security Event Token), we cover them below while showing full communication sequence/flow for each of them.
+We take the case of one transmitter and one receiver.
+## Poll Events Delivery
+![sse-events-poll](assets/sse-events-poll.svg)
+
+The transmitter has endpoints that respond to the requests sent by the receiver. And the receiver has a requester that sends requests.
+
+In case of a `verification` request, the `<verification_endpoint>` on the transmitter, acknowledges or returns an error to indicate, if the contents of the request were valid.
```

Comment:
```suggestion
In case of a `verification` request, the `<verification_endpoint>` on the transmitter acknowledges or returns an error to indicate if the contents of the request were valid.
```